lib/ms.js

```diff
@@ -23,6 +23,9 @@ function docpress (cwd, options) {
     .metadata(meta)
     .ignore('node_modules')
     .ignore(`!**/{${meta.docs}{,/**/*},*.md}`)
+    
+  if( typeof meta.ignore == 'object' && meta.ignore.forEach )
```

Comment:
Could you use {} around the body and remove the extra spaces in the ()? Just like the other code in this file.

Reply:
Also the space after `if`.

Reply:
Done